packages/rich-text/src/component/use-anchor-ref.js

```diff
@@ -31,8 +31,10 @@ export function useAnchorRef( { ref, value, settings = {} } ) {
 	const activeFormat = name ? getActiveFormat( value, name ) : undefined;
 
 	return useMemo( () => {
-		const { ownerDocument } = ref.current;
-		const { defaultView } = ownerDocument;
+		if ( ! ref.current ) return;
```

Comment:
When is this null? On first render?

Reply:
Seems like rewriting as a callback ref would fix it, this is typically the kind of bugs passing "ref" to hooks can create.

Reply:
(just a small remark, not necessarily for now)

Reply:
No. It's `undefined` when clicking the transform item.